x-pack/plugins/code/public/components/app.tsx

```diff
@@ -43,7 +42,7 @@ export const App = () => {
         <Route path={ROUTES.MAIN_ROOT} component={Main} />
         <Route path={ROUTES.ADMIN} component={Admin} />
         <Route path={ROUTES.SEARCH} component={Search} />
-        <Route path={ROUTES.REPO} render={Empty} exact={true} />
+        <Route path={ROUTES.REPO} component={Repo} exact={true} />
```

Comment:
what if a jump-to-definition goes to `/:resource/:org/:repo/${pathTypes}/:revision/:path*:goto(!.*)?` directly, which is a `ROUTES.MAIN` route?